core/kernel/initcall.c

```diff
@@ -25,8 +25,6 @@ void __weak call_initcalls(void)
 			     " failed", (vaddr_t)call - VCORE_START_VA);
 		}
 	}
-
-	IMSG("Initialized");
```

Comment:
May remove also the "Done" trace message from `call_finalcalls()`.

Reply:
Yes, I think I had removed it initially but the change was lost when rebasing. Thanks!